src/editor/components/scenegraph/Toolbar.js

```diff
@@ -182,25 +152,20 @@ export default class Toolbar extends Component {
         Events.emit('openpaymentmodal');
         return;
       }
-      let isCurrentUserTheSceneAuthor;
```

Comment:
no more network requests to see if the current user is the author. we already know that